chia/pools/pool_puzzles.py

```diff
@@ -160,10 +160,10 @@ def create_travel_spend(
         delay_ph,
     )
     if is_pool_member_inner_puzzle(inner_puzzle):
-        # inner sol is (spend_type, pool_reward_amount, pool_reward_height, extra_data)
-        inner_sol: Program = Program.to([1, 0, 0, bytes(target)])
+        # inner sol is key_value_list ()
+        inner_sol: Program = Program.to([bytes(target), ()])
```

Comment:
I recommend using `0` instead of `()` (in fact, I was unaware that `()` did anything in python).

Reply:
I just did `Program.to(())` and it gave me `ValueError: can't cast tuple of size 0` so I think this is unlikely to work properly.